conans/test/functional/layout/test_editable_cmake.py

```diff
@@ -21,13 +21,13 @@ def editable_cmake(generator):
            path=os.path.join(c.current_folder, "pkg"))
 
     def build_dep():
-        c.run("install . -if=install_release")
-        c.run("build . -if=install_release")
-        c.run("install . -s build_type=Debug -if=install_debug")
-        c.run("build . -if=install_debug")
+        c.run("install . -if=install_release --build-folder=mybase")
```

Comment:
Just to test, it works, this test probably needs to be parameterize, and leave the default case without args checking it works